pysatNASA/instruments/timed_guvi.py

```diff
@@ -59,20 +59,22 @@
 
 platform = 'timed'
 name = 'guvi'
-tags = {'': ''}
+tags = {'l1c_img': 'Level 1C imaging disk high resolution data',
+        'l1c2_img': 'Level 1C imaging disk low resolution data',
+        'l1c_spect': 'Level 1C spectrograh disk high resolution data',
+        'l1c2_spect': 'Level 1C spectrograph disk low resolution data'}
```

Comment:
I'd recommend simplifying these names a bit, primarily by removing the level designation from the tag string (but not the descriptions, those are excellent).  Perhaps something like "img_low", "img_high", 'spec_low', 'spec_high'?